__tests__/articles_comments-endpoint.test.js

```diff
@@ -0,0 +1,44 @@
+const data = require('../db//data/test-data')
+const request = require('supertest')
+const seed = require('../db/seeds/seed')
+const app = require('../app')
+const db = require('../db/connection')
+
+beforeEach(() => seed(data));
+afterAll(() => db.end());
+
+describe('Testing Articles Comments Endpoint', () => {
+    describe('/api/articles/:articles_id/comments', () => {
+        test('200: get 200 response', () => {
+            return request(app).get('/api/articles/1/comments').expect(200)
+        })
```

Comment:
Nice test, but you are making this test redundant by having all subsequent happy path tests have an expect(200) as well. I would recommend either removing all of those expects if you want to keep this test here